highfidelity/hifi
# Fix QML achoring warnings on Layouts

Test steps:

1.Run Interface
2. Check if there is no warnings about TabletHome anchoring

https://highfidelity.fogbugz.com/f/cases/20110/QML-prints-warnings-about-achoring
